tools/circle_plus_gen/how-to-write-tparam-json.md

```diff
@@ -0,0 +1,77 @@
+# How to write Hyperparameters JSON file
+
+The Hyperparameters JSON file is used to store hyperparameters that should be configured before training. For quicker understanding, please refer to the examples([#1](./example/train_tparam.json), [#2](./example/tparam_sgd_scce.json)) of the JSON file. 
+<br/>
+
+The json file consists of **a single JSON object** containing the following keys and corresponding values : 
+
+```json
+{
+  "optimizer" : {...}, 
+  "loss" : {...},
+  "batchSize" : 32, 
+}
+```
+
+- "optimizer" : refer [optimizer](#optimizer) for writing corresponding value
+- "loss" : refer [loss](#loss) for writing corresponding value
+- "batchSize" : a number of examples processeed during each iteration 
+
+
+## optimizer
+
+An object describing optimization algorithm. This should include two keys : 
+
+* `type` : a string to indiciate optimizer (e.g. `adam`) 
+* `args` : an object holding additional arguments to the chosen optimizer. These may vary depending on the optimizer type, but typically include 'learningRate'. 
```

Comment:
How about giving a link instead of explaining the details? This code looks very good, but I'm worried about maintaining it.

- link
  - Each namer from https://github.com/Samsung/ONE/blob/master/tools/circle_plus_gen/lib/utils.py
  - Others from https://github.com/Samsung/ONE/blob/master/tools/circle_plus_gen/schema/circle_traininfo_generated.py

Reply:
Sorry for being late.
I'm rewriting it as you suggested, now. 
I'll ask your review after updating this PR.

Reply:
@jyoungyun I tired another way. Could you take a look? 
Revision version might be easy to maintain. but a bit hard to understand at first glance.